Cargo.toml

```diff
@@ -103,7 +103,7 @@ members = [
 
 [workspace.dependencies]
 # rust-vmm crates
-acpi_tables = { git = "https://github.com/rust-vmm/acpi_tables", branch = "main" }
+acpi_tables = { git = "https://github.com/rust-vmm/acpi_tables", rev = "e08a3f0" }
```

Comment:
You shouldn't need to do this - just do `cargo update` to bump the revision in the `Cargo.lock`